lerobot/common/robot_devices/robots/factory.py

```diff
@@ -1,7 +1,9 @@
 import hydra
 from omegaconf import DictConfig
 
+from lerobot.common.robot_devices.robots.utils import Robot
 
-def make_robot(cfg: DictConfig):
+
+def make_robot(cfg: DictConfig) -> Robot:
```

Comment:
obvious no?

Reply:
I do type-hints mainly for easy access to the API on my IDE through completion ;)